src/textui.rst

```diff
@@ -459,8 +458,7 @@ dans le code suivant :
 
 ``--testdox``
 
-    Rapporte l'avancement des tests sous forme de documentation agile. Voir
-    :ref:`other-uses-for-tests` pour plus de détails.
+    Rapporte l'avancement des tests sous forme de documentation agile (Voir :ref:`textui.testdox`). Voir
```

Comment:
"the test progress as agile documentation" devient 
"the test progress in TestDox format"

https://github.com/sebastianbergmann/phpunit-documentation-english/commit/7903992eeabd6018c69c69f74c74d301ee9a0d30#diff-19c700533b36e5c238a20b56ed944996L462